node-operators/networks/full-node.md

```diff
@@ -70,11 +70,11 @@ First, we need to create a local directory to store the chain data and set the n
 
 ```
 mkdir {{ networks.moonbase.node_directory }}
+sudo chown -R $(id -u):$(id -g) {{ networks.moonbase.node_directory }}
```

Comment:
i like the one you had below better: `chown moonbase_service {{ networks.moonbase.node_directory }}`